tools/chpl-language-server/src/symbol_signature.py

```diff
@@ -76,23 +84,54 @@ def compute_type(self, via: Optional[chapel.TypedSignature] = None):
 
         Note: this is a temporary method that will go away when the resolver is fully online
         """
+        remove = []
         for i in range(len(self._signature)):
             tag = self._signature[i].tag
             node = self._signature[i].node
+            value = self._signature[i].value
+            prefix = self._signature[i].prefix or ""
+            postfix = self._signature[i].postfix or ""
```

Comment:
Could you do:
```Python
        for (i, part) in enumerate(self._signature):
            tag = part.tag
            node = part.node
            value = part.value
            prefix = part.prefix or ""
            postfix = part.postfix or ""
```

Then, I suggest in-lining the variables only used a handful of times (looks like `tag` and `value`)